token/program-2022-test/tests/confidential_transfer.rs

```diff
@@ -306,6 +306,75 @@ async fn confidential_transfer_configure_token_account() {
     );
 }
 
+#[tokio::test]
+async fn fail_approving_account_on_wrong_mint() {
```

Comment:
```suggestion
async fn confidential_transfer_fail_approving_account_on_wrong_mint() {
```
Just for consistency with the rest of the tests, should we append the `confidential_transfer_` prefix on the test names? This also makes filtering just the confidential transfer tests a little bit easier.

Reply:
Yeah sure that works for me! I typically avoid prefixing all the tests because the filename will do that prefixing for you, so I'll run `cargo test-sbf --test confidential_transfer -- fail_approve`, but I'll keep things consistent here